x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.test.tsx

```diff
@@ -45,13 +45,13 @@ describe('alert actions', () => {
     updateTimelineIsLoading = jest.fn() as jest.Mocked<UpdateTimelineLoading>;
 
     searchStrategyClient = {
+      ...dataPluginMock.createStartContract().search,
```

Comment:
Extending the search services triggered ts check error here. 
Reusing mock to avoid this in the future.